NAMESPACE

```diff
@@ -8,7 +8,6 @@ S3method(knit_print,CohortSizeOrdinal)
 S3method(knit_print,CohortSizeParts)
 S3method(knit_print,CohortSizeRange)
 S3method(knit_print,DADesign)
-S3method(knit_print,DataParts)
```

Comment:
is this ok to be removed?

Reply:
Nope.  It's not OK.  Typo on my part.  Will fix and resubmit.